src/TestSuite/IntegrationTestTrait.php

```diff
@@ -946,6 +946,7 @@ public function assertRedirectBack(string $message = ''): void
 
         $verboseMessage = $this->extractVerboseMessage($message);
         $this->assertThat(null, new HeaderSet($this->_response, 'Location'), $verboseMessage);
+        $this->assertThat(null, new StatusSuccess($this->_response), $verboseMessage);
```

Comment:
Technically we could also just say it has to be 302, since that is the only actual code here for such a redirect IMO.

Reply:
Another option would be to expose the error code as opt arg:

    assertRedirectBack(?int $code, string $message = '')
    
and defaulting to full range.

Reply:
> assertRedirectBack(?int $code, string $message = '')

I think that will work well.

Reply:
Done :)